src/main/java/com/pileproject/drivecommand/machine/device/input/simulation/SimulatedColorSensor.java

```diff
@@ -0,0 +1,38 @@
+package com.pileproject.drivecommand.machine.device.input.simulation;
```

Comment:
Hmm, I don't know which is better to be honest.
@tiwanari might have opinions on this.